test/e2e/specs/test.spec.js

```diff
@@ -35,6 +35,53 @@ describe('Wallet App Test Cases', () => {
       cy.get('span').contains('BLD').should('exist');
     });
 
+    it('should succeed in provisioning a new wallet ', () => {
+      const walletAddress = {
+        value: null,
+      };
+
+      cy.setupWallet({
+        createNewWallet: true,
+        walletName: 'user1',
+      });
+
+      cy.visit('/wallet');
+
+      cy.contains(/agoric.{39}/)
```

Comment:
what is the significance of 39 here? please document

Reply:
I think it would make more sense to be `/agoric1.{38/`. either way, worth pulling out as a constant. e.g. `AGORIC_ADDR_RE`